k8s/example/values.yaml

```diff
@@ -73,8 +73,8 @@ global:
     # PCS CF1/CF3 Cluster, whereas some can simply ignore the usage.
     resources:
       limits:
-        cpu: 100m
-        memory: 128Mi
+        cpu: 200m
```

Comment:
suggestion to leave the config set as-is solely for agent

Reply:
reverted